docs/section5/Rule5-44.md

```diff
@@ -0,0 +1,50 @@
+
+# Envelope - Rule 5-44  
+
+**Rule ID:** 5-44  
+**Rule Description:** Space Conditioning Categories. Space conditioning categories used to determine applicability of the envelope requirements in Tables G3.4-1 through G3.4-8 shall be the same as in the proposed design.
+
+Exception: Envelope components of the HVAC zones that are semiheated in the proposed design must meet conditioned envelope requirements in Tables G3.4-1 through G3.4-8 if, based on the sizing runs, these zones are served by a baseline system with sensible cooling output capacity 5 Btu/h·ft2 of floor area, or with heating output capacity greater than or equal to the criteria in Table G3.4-9, or that are indirectly conditioned spaces.  
+
+**Rule Assertion:** Baseline baseline equals proposed  
+**Appendix G Section:** Table G3.1 Section 5(b) Baseline  
+**Schema Version:** 0.0.39  
+
+**Applicability:** All required data elements exist for B_RMD  
+**Applicability Checks:** Buiding has spaces  
+
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+**Function Call:**
+
+  1. get_surface_conditioning_category()  
+
+## Applicability:
+- this rule applies to buildings with exterior or underground surfaces.
+- Get surface conditioning category dictionary for B_RMD: ```scc_dictionary_b = get_surface_conditioning_category(B_RMD)```
+- possible surface conditioning types are "UNREGULATED", "SEMI-EXTERIOR", "EXTERIOR MIXED", "EXTERIOR RESIDENTIAL", "EXTERIOR NON-RESIDENTIAL".  This rule applies if there is at least one surface that is one of: "SEMI-EXTERIOR", "EXTERIOR MIXED", "EXTERIOR RESIDENTIAL", "EXTERIOR NON-RESIDENTIAL"
+- Look at each surface id: ```for surface_id in scc_dictionary_b:```
+  - if the value is one of "SEMI-EXTERIOR", "EXTERIOR MIXED", "EXTERIOR RESIDENTIAL", "EXTERIOR NON-RESIDENTIAL", CONTINUE TO RULE LOGIC: ```if scc_dictionary_b[surface_id] in ["SEMI-EXTERIOR", "EXTERIOR MIXED", "EXTERIOR RESIDENTIAL", "EXTERIOR NON-RESIDENTIAL"]:```
+    - CONTINUE TO RULE LOGIC
+- if the applicability logic arrives here without going to rule logic, the rule is not applicable: RULE NOT APPLICABLE
+
+## Rule Logic:  
+
+- create a list of surfaces that do not comply with this rule: ```non_compliant_surface_ids = []```
+  
+- Get surface conditioning category dictionary for P_RMD: ```scc_dictionary_p = get_surface_conditioning_category(P_RMD)```  
+
+- For each building segment in the Baseline model: ```for building_surface_b in B_RMD...surfaces:```
```

Comment:
The description does not appear match the python. This is for each surface in the baseline model.